background/sync-manager.js

```diff
@@ -63,19 +63,19 @@ const syncMan = (() => {
       return status;
     },
 
-    async login(name = prefs.get('sync.enabled')) {
+    async login(name) {
       if (ready.then) await ready;
+      if (!name) name = prefs.get('sync.enabled');
```

Comment:
Why did you change this?

Reply:
Because `prefs` may be not ready. Look at git blame before asking.

Reply:
> Because prefs may be not ready.
This makes sense, I didn't know waiting for `ready` would solve it.

> Look at git blame before asking.
I spent a good time looking at the PR before posting my comments, and I may have asked unnecessary questions because I'm unfamiliar with the code. No need for hostility :) I was just trying to help with my time; I won't get in the way anymore. Cheers

Reply:
I'm not hostile per se, just implicitly unwelcoming to such kind of reviews.